docs/arcaflow_scenarios.md

```diff
@@ -0,0 +1,56 @@
+### Arcaflow Scenarios
+Arcaflow is a workflow engine in development which provides the ability to execute workflow steps in sequence, in parallel, repeatedly, etc. The main difference to competitors such as Netflix Conductor is the ability to run ad-hoc workflows without an infrastructure setup required.
+
+The engine uses containers to execute plugins and runs them either locally in Docker/Podman or remotely on a Kubernetes cluster. The workflow system is strongly typed and allows for generating JSON schema and OpenAPI documents for all data formats involved.
+
+#### Usage
+
+To enable arcaflow scenarios edit the kraken config file, go to the section `kraken -> chaos_scenarios` of the yaml structure
+and add a new element to the list named `arcaflow_scenarios` then add the desired scenario
+pointing to the `input.yaml` file.
+```
+kraken:
+    ...
+    chaos_scenarios:
+        - arcaflow_scenarios:
+            - scenarios/arcaflow/sysbench-cpu-hog/input.yaml
+```
+
+##### input.yaml
+The implemented scenarios can be found in *scenarios/arcaflow/<scenario_name>* folder.
+The entrypoint of each scenario is the *input.yaml* file. 
+In this file there are all the options to set up the scenario accordingly to the desired target 
+#### config.yaml
+The arcaflow config file. Here you can set the arcaflow deployer and the arcaflow log level.
+The supported deployers are:
```

Comment:
I would just add a point here to be sure to install/configure whatever deployer type you select from the below list